ui/src/components/HookForm/index.jsx

```diff
@@ -750,9 +750,7 @@ export default class HookForm extends Component {
                             window.env.TASKCLUSTER_ROOT_URL,
                             'reference/core/hooks/firing-hooks'
                           )}>
-                          {'"'}
-                          firing hooks
-                          {'"'}
+                          &apos;&quot;&apos; firing hooks &apos;&quot;&apos;
```

Comment:
I think you just want `&quot;` here (with no space)

Reply:
Yes

Reply:
Yes you are correct, my bad. Thanks